subsys/net/ip/6lo_private.h

```diff
@@ -81,6 +81,7 @@
 #define NET_6LO_NHC_UDP_PORT_10		0x02
 #define NET_6LO_NHC_UDP_PORT_11		0x03
 
+#define NET_6LO_NHC_UDP_CHECKSUM    0x04
```

Comment:
Please indent the value against other values, also the empty line before could be removed too.

Reply:
Done